src/test/kotlin/racingcar/domain/CarTest.kt

```diff
@@ -33,33 +15,32 @@ internal class CarTest {
 
     @Test
     fun `같은 거리인지 체크 다른 거리일 경우`() {
-        val car = Car(Name("test"))
-        val movedCar = car.move(Number(4))
+        val car = Car(Name("test"), Distance((1)))
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        val car = Car(Name("test"), Distance(1))
```